scripts/debug_tools/README.md

```diff
@@ -0,0 +1,32 @@
+These scripts may be used to aid debugging of clang crashes during analysis.
+The goal is to reproduce the crash locally, i.e. using the source files in the
+failure zip. During the process we take the original compile command database,
+compiler includes and compiler target files and we create the modified version
+of them (with `_DEBUG` postfix).  These modified files contain the paths
+prefixed with the root of the source directory which holds the dependent source
+files (`sources-root`).
+
+`prepare_analyzer_cmd.py` creates a new analyzer command which may be executed
+immediately, if cross translation unit (CTU) was disabled.  However, to debug
+CTU analysis related crashed in clang we have to synthetise the AST dump files
+too, so other phases are needed which are done in `prepare_all_cmd_for_ctu.py`.
+It executes `CodeChecker` with ctu-collect, therefore `CodeChecker` must be in
+the `PATH` and the proper venv must be set.
+
+Example session for debugging a clang crash:
+```
```

Comment:
If this is a shell script-like thing, we might want to turn on some syntax highlighting. Type `sh` after the three backticks.